lib/src/widgets/banner_widget.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import './vscout_logo_widget.dart';
+
+class BannerWidget extends StatelessWidget {
+  const BannerWidget({this.extraPadding});
+  // Compensate for top padding of element below by subtracting extraPadding.
+  final double extraPadding;
+
+  @override
+  Widget build(BuildContext context) {
+    final double logoSize = 48.0;
```

Comment:
does flutter automatically make apps responsive for different phone sizes or should we make a note to do that later?

Reply:
I haven't tried emulating a tablet, but I think it won't look horrible (like it'll be responsive, but maybe not optimal)